Add section to monitoring page about agent diagnostics (#2579)
elastic/observability-docs: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc

. Click **Apply changes** to apply the updated logging level to the agent.

[discrete]
[[collect-agent-diagnostics]]
= Collect {agent} diagnostics

{fleet} provides the ability to remotely generate and gather an {agent}'s diagnostics bundle.
An agent can gather and upload diagnostics if it is online in a `Healthy` or `Unhealthy` state.

> Diagnostics bundles can be downloaded from {fleet} to be viewed locally.

Reviewer: ```suggestion
To download the diagnostics bundle for local viewing:
```

. In {kib}, go to **Management -> {fleet} -> Agents**.

. In the **Host** column, click the agent's name.

. Click the **Diagnostics** tab.

> . Click the `Request diagnostics .zip` button.

Reviewer: ```suggestion
. Click the **Request diagnostics .zip** button.
```

+
[role="screenshot"]
image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agent details]

> Any in-progress or previously collected bundles for the {agent} will be listed below.

Reviewer: ```suggestion
Any in-progress or previously collected bundles for the {agent} will be listed on this page.
```

Reviewer: Generally a best practice to avoid using above, below, etc to describe areas of the UI.
